args/bosses.py

```diff
@@ -11,6 +11,8 @@ def parse(parser):
                         help = "Boss battles randomized")
     bosses.add_argument("-bmbd", "--mix-bosses-dragons", action = "store_true",
                         help = "Shuffle/randomize bosses and dragons together")
+    bosses.add_argument("-bmbs", "--mix-bosses-statues", action = "store_true",
+                        help = "Shuffle/randomize bosses and statues together")
```

Comment:
This flag seems to work more similarly to shuffle-random-phunbaba than to mix-bosses-dragons (if disabled they are not shuffled/randomized whereas if mix-bosses-dragons is disabled dragons can still be shuffled/randomized amongst themselves). Should it be called shuffle-random-statues? The answer to this may also depend on the answer to the other question of random/fixed encounters.

Reply:
Good point here. Perhaps we could add a `shuffle dragons` and `shuffle statues` option on top of this?